src/Microsoft.TemplateEngine.Edge/Installers/NuGet/NugetApiPackageManager.cs

```diff
@@ -346,7 +382,7 @@ private IEnumerable<PackageSource> LoadNuGetSources(params string[] additionalSo
                 customSources.Add(packageSource);
             }
 
-            IEnumerable<PackageSource> retrievedSources = defaultSources.Concat(customSources);
+            IEnumerable<PackageSource> retrievedSources = customSources.Concat(defaultSources);
```

Comment:
Is this just about perf or also changes behavior? Wondering how hard would be to unit test...

Reply:
it doesn't change the behavior; in fact with parallelization everywhere the impact is minimum.

The idea behind it is that if the user specifies the source the package is likely to be found in that source, and not the default ones. Previously checking for the packages was sequential until the package is found, so it mattered. Now that processing is parallel it doesn't matter that much.